code/modules/surgery/plastic_surgery.dm

```diff
@@ -9,6 +14,41 @@
 		/datum/surgery_step/close,
 	)
 
+/datum/surgery/plastic_surgery/advanced
+	name = "advanced plastic surgery"
+	steps = list(
+		/datum/surgery_step/incise,
+		/datum/surgery_step/retract_skin,
+		/datum/surgery_step/insert_plastic,
+		/datum/surgery_step/reshape_face,
+		/datum/surgery_step/close,
+	)
+
+//insert plastic
```

Comment:
:(

Reply:
I have zero idea what you mean :(
I added a happy face in the comment to compensate :)

Reply:
Aight fair enough